model_package_core/constants.py

```diff
@@ -3,3 +3,10 @@
 PEER_MODEL_PATH = Path('.', 'peers', 'models')
 WEIGHT_FILENAME = 'weights.h5'
 METADATA_FILENAME = 'metadata.json'
+
+# base path of the data
+DATA_PATH = Path('model_package_core', 'data')
+
+# train data path
+TRAIN_IMAGE_DATA_PATH = Path(DATA_PATH, 'train/MNIST/raw/train-images-idx3-ubyte')
```

Comment:
This is not ideal because it now requires training data to be committed to the repo to run any tests. I think it would be better for these constants to read from environment variables. We can discuss this more over our next call if you'd like!

Reply:
Let's go over this in the call!